komga-webui/src/components/dialogs/EditSeriesDialog.vue

```diff
@@ -611,6 +659,40 @@ export default Vue.extend({
       return null
     },
     async editSeries(): Promise<boolean> {
+      if (this.single && this.uploadQueue.length > 0) {
+        const series = this.series as SeriesDto
+        let hadErrors = false
+        for (const file of this.uploadQueue) {
+          try {
+            const result = await this.$komgaSeries.uploadThumbnail(series.id, file, file.name === this.selectedThumbnail)
+            if (result.violations) {
+              for (const violation of result.violations) {
+                this.$eventHub.$emit(ERROR, {message: violation.message} as ErrorEvent)
```

Comment:
in which cases would you get a violation message ? the endpoint doesn't have any validation rules, so it would not throw any